src/tools/pub/glossary.md

```diff
@@ -29,6 +29,74 @@ specify `any` for their dependencies' [version constraints][].
 [lockfiles]: #lockfile
 [version constraints]: #version-constraint
 
+## Content hashes
+
+The pub.dev repository serves a sha256 hash of each package version it hosts.
```

Comment:
**question:** "Serves" reads odd here. How is a hash served? Is it listed with the package? Is it included as part of the download? Also, looking to the next sentence, make it clear that `pub` is doing this validation, that is, The `pub` client validates a package's integrity using this hash.

Reply:
>  Also, looking to the next sentence, make it clear that pub is doing this validation, that is, The pub client validates a package's integrity using this hash.

The second sentence already says literally that exact same thing: 

> "The pub client can use this hash to validate the integrity of downloaded packages,
> and protect against changes on the repository." 

So I'm not sure what you're asking for there.